core/server/src/main/java/alluxio/master/file/FileSystemMasterClientServiceHandler.java

```diff
@@ -147,7 +147,13 @@ public FileInfo getStatusInternal(long fileId) throws AlluxioTException {
     }
   }
 
+  /**
+   * {@inheritDoc}
+   *
+   * @deprecated since version 1.1 and will be removed in version 2.0
```

Comment:
there is no alternative ... this method should simply not be used, because it allows the client to write code that assuming that there is only one underFS, which is something we want to prevent from happening